scripts/qesap/test/unit/test_qesap_lib_cmd_ansible.py

```diff
@@ -0,0 +1,37 @@
+from unittest import mock
+import logging
+import os
+
+
+from lib.cmds import export_ansible_output
+
+log = logging.getLogger(__name__)
+
+
+def test_export_ansible_output():
+    '''
+    This test checks if the ansible execution generates log files
+    in this case generates the generic one "ansible.testAll.log.txt"
+    scripts/qesap/test/unit/test_qesap_ansible.py
+    
+    '''
+
+    test_dir = os.getcwd()
+    test_file = os.path.join(test_dir, 'ansible.testAll.log.txt')
+    command_to_sent = {'test1':'test_value1', 'test2':'test_value2', 'test3':'test_value3', 'test4':'test_value4',
+                        'cmd':['/tmp/exec_venv/bin/ansible', '-vvvv', '-i', '/root/qe-sap-deployment/terraform/aws/inventory.yaml',
+                              'testAll', '-a', '--ssh-extra-args="-l cloudadmin -o UpdateHostKeys=yes -o StrictHostKeyChecking=accept-new"']}
+    ansible_output = """DEBUG    OUTPUT: ansible [core 2.13.5]
+                        DEBUG    OUTPUT:   config file = None
+                        DEBUG    OUTPUT:   configured module search path = ['/root/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
+                        DEBUG    OUTPUT:   ansible python module location = /tmp/exec_venv/lib64/python3.11/site-packages/ansible
+                        DEBUG    OUTPUT:   ansible collection location = /root/.ansible/collections:/usr/share/ansible/collections
+                        DEBUG    OUTPUT:   executable location = /tmp/exec_venv/bin/ansible
+                        DEBUG    OUTPUT:   python version = 3.11.5 (main, Sep 06 2023, 11:21:05) [GCC]
+                        DEBUG    OUTPUT:   jinja version = 3.1.4
+                        DEBUG    OUTPUT:   libyaml = True"""
+
+    export_ansible_output(command_to_sent, ansible_output)
+
+    assert os.path.isfile(test_file), f"Ansible output file {test_file} was not created."
+    os.remove(test_file)
```

Comment:
it is ok that you remove the file after the test. Generally speaking I do no t like when a UT write file around. I like to try to always let the UT to confine the place to read and write the FS to https://docs.pytest.org/en/7.1.x/how-to/tmp_path.html

This comment is a generic suggestion and we cal look at this later.